test/extended/openstack/loadbalancers.go

```diff
@@ -136,6 +142,14 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack][lb] The Openstack
 			e2e.Logf("Pods accessed after 100 UDP requests:\n%v\n", results)
 			pods, err := oc.KubeClient().CoreV1().Pods(oc.Namespace()).List(context.Background(), metav1.ListOptions{})
 			o.Expect(err).NotTo(o.HaveOccurred())
+
+			// ETP:Local not configured so the defined lbMethod is not applied
+			// https://issues.redhat.com/browse/OCPBUGS-2350
+			if lbProviderUnderTest == "OVN" {
+				lbMethod = "source_ip_port"
```

Comment:
is `lbMethod` different from what is expected [on line 125](https://github.com/openshift/openstack-test/pull/68/files#diff-3c085086ae67274b973d70d10a2e27a2d4947730b75e358271611784b2d9b181R125)? If not, you should probably move this code above (there is no need to gather the lb method with getClusterLoadBalancerSetting() if we know what it should be), otherwise, if they serve different purpose I suggest you use a different variable name.

Reply:
it serves different purpose: lbMethod is the value configured on OCP, and we compare it to what is present on the OSP LB method.

This change is different purpose, so I am changing the name. Thanks